scripts-dev/proxy-test/docker-compose.yml

```diff
@@ -11,18 +13,28 @@ services:
       - ./curl.sh:/curl.sh
       - ./notification-gcm.json:/notification-gcm.json
       - ./notification-ios.json:/notification-ios.json
-      - ./proxy.conf:/etc/apt/apt.conf.d/proxy.conf
+      - ./mitmproxy:/mitmproxy:ro
+    environment:
+      SSL_CERT_FILE: /mitmproxy/ca.crt
     ports:
       - 5000:5000
 
   proxy:
-    image: dominikbechstein/nginx-forward-proxy
+    image: docker.io/mitmproxy/mitmproxy
+    command: >-
+      mitmdump
+      -w /out/mitmdump_out
+      --map-local '|https://oauth2.googleapis.com/token|/responses/oauth2_googleapis_token.json'
+      --map-local '|https://fcm.googleapis.com/v1/projects/*|/responses/fcm_push.json'
     networks:
       no-internet:
         ipv4_address: 172.28.0.3
       internet:
-    container_name: nginx-forward-proxy
+    container_name: mitmdump
     volumes:
+      - ./out:/out:Z
+      - ./mitmproxy:/home/mitmproxy/.mitmproxy:z
+      - ./responses:/responses:ro
       - ./nginx.conf:/usr/local/nginx/conf/nginx.conf:ro
```

Comment:
I assume this and the `/scripts-dev/proxy-test/nginx.conf` file can also be removed?
```suggestion
```